tox.ini

```diff
@@ -60,7 +60,7 @@ commands =
     format: isort --quiet --atomic viahtml tests bin
     checkformatting: black --check viahtml tests bin
     checkformatting: isort --quiet --check-only viahtml tests bin
-    tests: coverage run -m pytest {posargs:tests/unit/}
+    tests: coverage run -m pytest {posargs:tests/}
```

Comment:
Run the lot. There's not point in doing it separately at the moment. Also the functional tests are faster in this case

Reply:
I think we should change this to be the same as our other apps in some ways:

1. `make test` should run the unit tests only
2. There should be a separate `make functests` command to run the functional tests
3. Only the unit tests should count towards test coverage, not the functional tests

This is in line with our other apps, and coverage isn't a very useful signal with functional tests. I can imagine functional test coverage being difficult to maintain, too. When you have a gap in `viahtml/foo.py`'s unit test coverage it's relatively easy to add another test to `tests/unit/viahtml/foo_test.py` (or to find the existing `foo_test.py` test that should be covering the missed lines but isn't working). But if a change causes the _functional_ tests to have a gap in coverage of a certain source file, I can imagine it being relatively difficult to figure out what the problem is and how to fix it, whether there's a test that supposed to be covering these lines but isn't because it's misbehaving and if so which test and how to fix it, or whether there's a missing test that should be added. It's just a lot harder to know exactly what lines are / should be covered under what test(s). You lack the 1:1 relationship between source module and test module and so much more code is being executed under each test, so much more is going on under each test, that it's much harder to understand.

Of course it's going to be impossible to get 100% test coverage out of viahtml by use of unit tests only, due to the nature of viahtml and its relationship to pywb. I think the solution to this should be the same solution that we apply to our other apps when we have some code that is too difficult or not worthwhile to unit test: use `# pragma: nocover` comments and/or ignore entire files in `.coveragerc`. The difference will be in degree: viahtml will make a lot more use of coverage ignores.

The coverage ignores might function as handy warning signs also: warning developers when they're touching parts of viahtml that don't have unit test coverage.